spark/src/main/scala/io/delta/sharing/spark/DeltaSharingSource.scala

```diff
@@ -560,7 +612,12 @@ case class DeltaSharingSource(
       idToUrl,
       Seq(new WeakReference(fileIndex)),
       params.profileProvider,
-      latestRefreshFunc
+      latestRefreshFunc,
+      if (minUrlExpirationTimestamp != -1) {
+        minUrlExpirationTimestamp
+      } else {
+        lastQueryTableTimestamp + CachedTableManager.INSTANCE.preSignedUrlExpirationMs
```

Comment:
Should we use refreshThresholdMs in this case as before?

Reply:
Could you elaborate a bit?

`refreshThresholdMs` is the buffer time before expiration for the refresh, how is it related to the value here?